code/modules/games/cards.dm

```diff
@@ -214,7 +224,7 @@
 	if(!discarding || !to_discard[discarding] || !usr || !src) return
 
 	var/datum/playingcard/card = to_discard[discarding]
-	qdel(to_discard)
+	to_discard.Cut()
```

Comment:
@PsiOmegaDelta 

Are you completely positive about that? I thought lists were finally moved to be a child of datum?